app/src/main/java/com/appunite/loudius/domain/repository/AuthRepository.kt

```diff
@@ -19,8 +19,6 @@ package com.appunite.loudius.domain.repository
 import com.appunite.loudius.domain.store.UserLocalDataSource
 import com.appunite.loudius.network.datasource.AuthDataSource
 import com.appunite.loudius.network.model.AccessToken
-import javax.inject.Inject
-import javax.inject.Singleton
```

Comment:
Question: Is it really building in the correct way? We have annotations below and here are deleted imports ;)

Reply:
should be good now :)